README.rst

```diff
@@ -248,20 +253,44 @@ To simplify data conversion from Spark to TensorFlow, you can use Spark converte
     # make a tensorflow dataset from `converter1
     with converter1.make_tf_dataset() as dataset:
         # the `dataset` is `tf.data.Dataset` object
-        # we can train/evaluate model on `dataset`
-        # when exit the with context, the reader of the dataset will be closed
-        ...
+        # dataset transformation can be done if needed
+        dataset = dataset.map(...)
+        # we can train/evaluate model on the `dataset`
+        model.fit(dataset)
+        # when exiting the context, the reader of the dataset will be closed
 
     # delete the cached files of the dataframe.
     converter1.delete()
 
+The minimalist example below assumes the definition of a ``Net`` class and
+``train`` and ``test`` functions, and a Spark DataFrame containing a feature
+column followed by a label column.
+
+.. code-block:: python
+
+    from petastorm.spark import SparkDatasetConverter, make_spark_converter
+
+    spark.conf.set(SparkDatasetConverter.PARENT_CACHE_DIR_URL_CONF, 'hdfs:/...')
+
+    df1 = ... # `df1` is a spark dataframe
+    model = Net()
```

Comment:
What's the Net ? An empty model ?

Reply:
I will mention in the description: "The minimalist example below assumes the definition of a ``Net`` class and
``train`` and ``test`` functions, included in ``pytorch_example``, ..."